packages/odyssey-react/src/components/Tag/Tag.theme.ts

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  // Color
+  TagBackgroundColor: theme.ColorPaletteNeutral100,
```

Comment:
We somehow just talked about this during office hour. 

1. Looking at other examples, some reducer commit having the "components" name in the variable name, some have it at the beginning, others at the end.
2. Should we attempt to have combine variables when possible or is having 1 PER usage the correct approach?

Reply:
great questions:

1. there may be some outliers here but the goal is to never have the component name in a variable name. See my above comment around why. What you're seeing could be component parts such as labels or content areas or other bits and pieces. We also have a mixed implementation today around start or end, but for now favor what feels right, most folks @andrewberg-okta and @edburyenegren-okta like having those at the start.
2. Only combine variables if they have the same functional purpose. If the margin and padding styles both have the same value, define **separate** component theme variables, even if they are mapped to the same global space token. This will provide a more natural API for downstream consumers.

As an example of when to share, in the case of RTL or LTR specific styles (when both are present) you **should** share a single variable that has the same function. Even if it happens to be applied to different style declarations! This will feel most natural from a downstream develop's perspective. 

cc @edburyenegren-okta @andrewberg-okta